shell/common/animator.cc

```diff
@@ -74,13 +76,27 @@ void Animator::SetDimensionChangePending() {
   dimension_change_pending_ = true;
 }
 
-void Animator::EnqueueTraceFlowId(uint64_t trace_flow_id) {
+void Animator::EnqueueTraceData(const PointerDataPacket& packet,
+                                uint64_t trace_flow_id) {
+  std::vector<PointerData> pointer_datas;
+  constexpr size_t kBytesPerPointerData =
+      kPointerDataFieldCount * kBytesPerField;
```

Comment:
We have`static_assert(sizeof(PointerData) == kBytesPerField * kPointerDataFieldCount)` in https://github.com/flutter/engine/blob/master/lib/ui/window/pointer_data.cc#L11  so we can just use `sizeof(PointerData)` here.